server/README.md

```diff
@@ -331,8 +331,13 @@ cargo run --bin remote_server_cli -- initialise-from-export -n 'export_name' -r
 cargo run --bin remote_server_cli -- initialise-from-central -u 'user1:password1'
 # attempt to refresh dates (advance them forward, see --help)
 cargo run --bin remote_server_cli -- refresh-dates
+# See README.md link below
+cargo run --bin remote_server_cli -- backup
+cargo run --bin remote_server_cli -- restore D2024_08_22T05_05_16 -s
```

Comment:
```suggestion
cargo run --bin remote_server_cli -- restore --backup-name D2024_08_22T05_05_16 -s
```

or `-b` probs worth mentioning either is possible

Reply:
Thanks updated with `-b` and also the `--help` should show full description (short and long form)